src/pages/Home.jsx

```diff
@@ -125,37 +143,42 @@ const Home = (props) => {
-                    <p style={paragraph}>To better enable the use of existing human subjects datasets in future projects, DUOS mimics, in a semi-automated fashion, the data access request review processes common to DACs globally, like those in dbGaP. To this end, the system includes interfaces to capture and structure data use restrictions and data access requests as machine-readable data use terms based on the GA4GH&apos;s Data Use Ontology. With these machine-readable terms for dataset&apos;s use limitations and data access requests established, DUOS is able to trigger a matching algorithm to reason if data access should be granted given the research purpose and the data restrictions, serving as a decision support tool for DACs using DUOS.</p>
-                    <p style={paragraph}>To evaluate the feasibility of using machine readable data use terms to interpret data use restrictions and access requests, we are piloting a trial of DUOS overseen by Partners’ Healthcare IRB. During the pilot, DACs comprised of governmental and non-governmental data custodians will pilot the use of DUOS, its ability to structure use limitations and access requests, and the accuracy of the DUOS algorithm. This aids us in improving the DUOS algorithm and providing feedback on the GA4GH Data Use Ontology based on experts’ feedback.</p>
+
+        <section style={{ margin: '5rem auto', padding: '0 2rem' }}>
+          <div style={{ maxWidth: '1200px', margin: '0 auto', textAlign: 'center' }}>
+            <p style={header}>Data Libraries in DUOS</p>
+            <p style={description}>Click the images below to view curated Data Libraries, and search and request access to data.</p>
+
+            <div style={logoGrid} className="logo-grid">
+              <Tooltip title="AnVIL" placement="bottom" arrow>
+                <a href="https://duos.org/datalibrary/anvil" target="_blank" rel="noreferrer" style={{ textDecoration: 'none' }}>
+                  <div className="logo-card" style={baseCard}>
+                    <img src={anvilLogo} alt="AnVIL" style={logoImg} />
                   </div>
-                ]}
-              />
+                </a>
+              </Tooltip>
+
+              <Tooltip title="Broad Institute" placement="bottom" arrow>
+                <a href="https://duos.org/datalibrary/broad" target="_blank" rel="noreferrer" style={{ textDecoration: 'none' }}>
+                  <div className="logo-card" style={{ ...baseCard, background: '#1F3B50', padding: '15px' }}>
+                    <img src={broadLogo} alt="Broad Institute" style={logoImg} />
+                  </div>
+                </a>
+              </Tooltip>
+       
+              <Tooltip title="Human Cell Atlas" placement="bottom" arrow>
```

Comment:
The tooltip text here is very difficult to read. Is there anything we can do to increase the font styling?
![Screenshot 2025-05-13 at 8 44 55 AM](https://github.com/user-attachments/assets/9a2c877c-163e-42b4-957c-7734fc76b59e)